tests/test_lookups.py

```diff
@@ -84,24 +130,22 @@ def test_localized_string_lookups_n(self):
             ("string_n__any_lang_icontains", "ORTY-THREE"),
         ]:
             with self.subTest(lookup=lookup, value=value):
-                self.assertFalse(resources.filter(**{lookup: value}))
+                self.assertFalse(self.resources.filter(**{lookup: value}))
 
-    def test_resource_instance_lookups(self):
-        resources = ResourceTileTree.get_tiles("datatype_lookups")
 
+class ResourceInstanceLookupTests(GenericLookupTests):
+    def test_cardinality_1(self):
         for lookup, value in [
             ("resource_instance__id", str(self.resource_42.pk)),
             ("resource_instance_list__contains", str(self.resource_42.pk)),
         ]:
             with self.subTest(lookup=lookup, value=value):
-                self.assertTrue(resources.filter(**{lookup: value}))
-
-    def test_resource_instance_lookups_n(self):
-        resources = ResourceTileTree.get_tiles("datatype_lookups")
+                self.assertTrue(self.resources.filter(**{lookup: value}))
 
+    def test_cardinality_n(self):
         for lookup, value in [
             ("resource_instance_n__ids_contain", str(self.resource_42.pk)),
-            # ("resource_instance_list_n__ids__contain", str(self.resource_42.pk)),
+            ("resource_instance_list_n__ids_contain", str(self.resource_42.pk)),
         ]:
             with self.subTest(lookup=lookup, value=value):
-                self.assertTrue(resources.filter(**{lookup: value}))
+                self.assertTrue(self.resources.filter(**{lookup: value}))
```

Comment:
The intent is for these tests to show what kinds of "better" lookups are possible versus the "plain" ones you get in the `GenericLookupTests`. I'd like to add some for concept values, resource display names, etc., but I had to put this down for the moment.